File: programacao/c/respostas/selecao/33.c
#include <stdio.h>

int main(void) {
    int lados;

    printf("Digite o numero de arestas: ");
    scanf("%d", &lados);

    if (lados < 3 || lados > 10) {
        printf("Erro\n");
    } else if (lados == 3) {
        printf("Triangulo\n");
    } else if (lados == 4) {
        printf("Quadrilátero\n");
    } else if (lados == 5) {
        printf("Pentagono\n");
    } else if (lados == 6) {
        printf("Hexagono\n");
    } else if (lados == 7) {
        printf("Heptagono\n");
    } else if (lados == 8) {
        printf("Octogono\n");
    } else if (lados == 9) {
        printf("Eneagono\n");
    } else {
        printf("Decagono\n");
    }

    return 0;
}

File: programacao/python/respostas/selecao/33.py
lados = int(input("Digite o numero de arestas: "))

if lados < 3 or lados > 10:
    print("Erro")
elif lados == 3:
    print("Triangulo")
elif lados == 4:
    print("Quadrilátero")
elif lados == 5:
    print("Pentagono")
elif lados == 6:
    print("Hexagono")
elif lados == 7:
    print("Heptagono")
elif lados == 8:
    print("Octogono")
elif lados == 9:
    print("Eneagono")
else:
    print("Decagono")
